### PR TITLE
update david's bio

### PR DESCRIPTION
**Pull request by dlebauer**
Also added logo, configured some settings

### CODE REVIEW

**Comment by MagicMilly** [comment]
Not sure if you addressed this or not with the widgets / configuration, but the top menu links are all working except for `People`, which still wants to be a projects section. 
Screen Shot 2020-05-29 at 11 32 48 AM
Screen Shot 2020-05-29 at 11 32 05 AM

File: content/people/david-lebauer/index.md [hunk 1]
```diff
@@ -27,4 +27,11 @@ caption = ""
 
 +++
 
-David steers this ship
\ No newline at end of file
+
+Bioscience Research Labs Room 207
```
Comment (KristinaRiemer) [inline content/people/david-lebauer/index.md:31]: This is redundant with what's in the contact section on the main page? If this contact info is included on David's, everyone's page should have it.  

It's also not formatted correctly: 
Screen Shot 2020-06-26 at 11 32 16 AM

**Comment by KristinaRiemer** [comment]
Piggybacking off of @MagicMilly's comment, any time I click on a section, the url says `projects` at the end no matter what. 

Screen Shot 2020-06-26 at 11 47 39 AM

**Comment by KristinaRiemer** [comment]
@dlebauer I think it would be easiest for now to just roll back the changes to people.md and projects.md. I know it's not ideal to have the people section stuff within a file called projects, but it would work for now. We can put fixing that on our to-do list. 

I based this on the Weecology website. They've managed to [rename the markdown to people.md](https://github.com/weecology/website/blob/master/content/home/people.md), so we would just need to figure that out.

**Comment by KristinaRiemer** [comment]
Where does [the plot png](https://github.com/az-digitalag/group-website/blob/7e60057d0b31a93fa40802d9105d26d9f3936b9e/static/img/green_field_heatmap.png) show up on the website? I can't find it.

**Pull request merged**